src/test/e2e/api/admin/event.e2e.test.ts

```diff
@@ -90,75 +90,6 @@ test('Can filter by project', async () => {
         });
 });
 
-test('can search for events', async () => {
-    const events: IBaseEvent[] = [
-        {
-            type: FEATURE_CREATED,
-            project: randomId(),
-            data: { id: randomId() },
-            tags: [],
-            createdBy: randomId(),
-            createdByUserId: TEST_USER_ID,
-            ip: '127.0.0.1',
-        },
-        {
-            type: FEATURE_CREATED,
-            project: randomId(),
-            data: { id: randomId() },
-            preData: { id: randomId() },
-            tags: [{ type: 'simple', value: randomId() }],
-            createdBy: randomId(),
-            createdByUserId: TEST_USER_ID,
-            ip: '127.0.0.1',
-        },
-    ];
-
-    await Promise.all(
-        events.map((event) => {
-            return eventService.storeEvent(event);
-        }),
-    );
-
-    await app.request
-        .post('/api/admin/events/search')
-        .send({})
-        .expect(200)
-        .expect((res) => {
-            expect(res.body.events).toHaveLength(2);
-        });
-    await app.request
-        .post('/api/admin/events/search')
-        .send({ limit: 1, offset: 1 })
-        .expect(200)
-        .expect((res) => {
-            expect(res.body.events).toHaveLength(1);
-        });
-    await app.request
-        .post('/api/admin/events/search')
-        .send({ query: events[1].data.id })
-        .expect(200)
-        .expect((res) => {
-            expect(res.body.events).toHaveLength(1);
-            expect(res.body.events[0].data.id).toEqual(events[1].data.id);
-        });
-    await app.request
-        .post('/api/admin/events/search')
-        .send({ query: events[1].preData.id })
-        .expect(200)
-        .expect((res) => {
-            expect(res.body.events).toHaveLength(1);
-            expect(res.body.events[0].preData.id).toEqual(events[1].preData.id);
-        });
-    await app.request
-        .post('/api/admin/events/search')
-        .send({ query: events[1].tags![0].value })
-        .expect(200)
-        .expect((res) => {
-            expect(res.body.events).toHaveLength(1);
-            expect(res.body.events[0].data.id).toEqual(events[1].data.id);
-        });
-});
-
```

Comment:
Should be covered by [event-search.e2e.test.ts](https://github.com/Unleash/unleash/blob/main/src/test/e2e/api/admin/event-search.e2e.test.ts)